ploneorg/theme/theme/plone-open-garden.html

```diff
@@ -2295,6 +2272,10 @@ <h2>Being guests at Hotel Mediterraneo.</h2>
           Sorrento Peninsula, a few minutes’ walk from the city center and a few kilometers away from the archeological
           sites of Ercolano and Pompei; and from there you can reach the island of Capri in just 15 minutes by boat.
       </p>
+      <p>We have obtained discounted prices for PLOG. There is a <a href="https://plone.org/events/sprints/plog-2020">
+	  range of accomodation options available</a>, depending on the type of room you choose. 
+          This offer is valid on a limited number of allocated rooms, if reserved before February 28, 2020.
```

Comment:
You could mention the deadline but link to the relevant anchor on the event.

Reply:
There isn't an anchor on the deadline, what anchor are you referring to?

Reply:
...well, you *could* manually add an anchor on the event where you state the deadline...

or (simpler) mention that the deadline is listed on the event page (and include a link to the event)

Reply:
The deadline is listed at the top of both event and registration form, so I didn't worry about anchors. I did add a sentence and link about details being on the event.